client/src/components/AuthFooter/AuthFooter.tsx

```diff
@@ -7,22 +6,24 @@ import { Typography } from '@material-ui/core';
 interface Props {
   linkTo: string;
   asideText: string;
-  btnText: string;
+  linkText: string;
 }
 
-const AuthHeader = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
+const AuthFooter = ({ linkTo, asideText, linkText }: Props): JSX.Element => {
   const classes = useStyles();
 
   return (
     <Box p={1} className={classes.authHeader}>
-      <Typography className={classes.accAside}>{asideText}</Typography>
+      <Typography className={classes.accAside} style={{ display: 'inline' }}>
```

Comment:
Try to avoid inline styles, you can put this styling in a class instead

Reply:
Make sure to address every component w/ inline styling

Reply:
fixed